### PR TITLE
INREL-7226 nexx

### PR DESCRIPTION
**Pull request by Nicozilla**
## [INREL-7226](https://jira.burda.com/browse/INREL-7226) 
- nexx video playlist is integrated above tags of articles 
- design aspects are considered
- video starts autoplay when it is in viewport
- on native articles playlist is not shown
- implementation considers amp templates and non amp templates

## How to test:
https://feature2.elle.de/trend-frisuren-offene-haare

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [ ] I have [left the code in a better state](http://programmer.97things.oreilly.com/wiki/index.php/The_Boy_Scout_Rule)
- [ ] I have documented the changes (where applicable)

### CODE REVIEW

File: js/behaviors/drupal.behaviors.videos.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    inviewObserver: null,
+    videoModel: null,
+    initialize: function(options) {
+      BaseInviewView.prototype.initialize.call(this, options);
+      this.id = this.videoModel.get('containerId');
+      this.delegateInview();
+    },
+    delegateInview: function() {
+      this.inviewObserver = new IntersectionObserver(
+        this.handleInviewObserver.bind(this)
```
Comment (danschalow) [inline js/behaviors/drupal.behaviors.videos.js:14]: Is this `bind` necessary?

File: sass/components/_media.scss [hunk 2]
```diff
@@ -49,6 +50,50 @@
...
+  position: relative;
+}
+
+#nexx-video--article-footer-wrapper {
+  .item-paragraph__title {
+    margin-bottom: $grid-unit-2;
+    font-size: 30px;
+  }
+}
+
```
Comment (danschalow) [inline sass/components/_media.scss:56]: Have we got variables for the above magic numbers?

**Review by danschalow — CHANGES_REQUESTED** [review]

File: js/behaviors/drupal.behaviors.videos.js (hunk 1 above)
Comment (Nicozilla) [reply js/behaviors/drupal.behaviors.videos.js:14]: yep. lost scope in IntersectionObserver

**Review by danschalow — APPROVED** [review]
LGTM

**Pull request merged**